common/src/main/java/gov/cms/ab2d/common/config/Mapping.java

```diff
@@ -50,8 +51,11 @@ public void init() {
                                 context.getSource().getAttestedOn().toString());
         Converter<ContractDTO, Contract> sponsorDTOSponsorConverter = new AbstractConverter<>() {
             protected Contract convert(ContractDTO source) {
+                Optional<Contract> contract = contractService.getContractByContractNumber(source.getContractNumber());
                 //noinspection OptionalGetWithoutIsPresent
-                return contractService.getContractByContractNumber(source.getContractNumber()).get();
+                if (contract.isPresent())
+                    return contract.get();
+                return null;
```

Comment:
We have already flagged this with /noinspection.  The code is structured as intended.  Put a line in so the SonarCube ignores this line.  No reason to complicated the code for a case that never will happen.